clang-tools-extra/clangd/unittests/RenameTests.cpp

```diff
@@ -926,12 +975,12 @@ TEST(RenameTest, Renameable) {
         void f(X x) {x+^+;})cpp",
           "no symbol", HeaderFile},
 
-      {R"cpp(// disallow rename on non-normal identifiers.
+      {R"cpp(
          @interface Foo {}
-         -(int) fo^o:(int)x; // Token is an identifier, but declaration name isn't a simple identifier.
+         -(int) [[fo^o]]:(int)x;
          @end
        )cpp",
-       "not a supported kind", HeaderFile},
+       nullptr, HeaderFile},
```

Comment:
Why's the error lost?

Reply:
That’s the whole purpose of the PR. We used to error because we couldn’t rename Obj-C symbols and now we can. So there’s no error anymore.

Reply:
Must have been tired when I was reviewing this, I think I saw "fo^o" and thought "huh, that should still error". But that's just the location 😅